chapters/object_types/frame.txt

```diff
@@ -75,13 +75,21 @@ access the stored pixel information – via
 
 [source,cpp]
 ....
-const void *anariMapFrame(ANARIDevice, ANARIFrame, const char *channel);
+const void* anariMapFrame(ANARIDevice,
+                          ANARIFrame,
+                          const char* channel,
+                          uint32_t *width,
+                          uint32_t *height,
+                          ANARIDataType *type);
```

Comment:
Better/needed to name it `pixelType`? We have `elementType` for arrays.

Reply:
Addressed in latest push.